### DISCUSSION TITLE
How to "Leave a Comment" in Channel?

### DISCUSSION THREAD

**Discussion by KillWolfVlad** [telegraf/telegraf#1895] [General]
Hi! I want to create an alert bot that send message with alert to Channel and send some additional details in Comments. How I can do that?

P.S. I found similar question, but without answer https://github.com/telegraf/telegraf/discussions/1875.

<img width="434" alt="image" src="https://github.com/telegraf/telegraf/assets/18448635/667a0091-a303-4942-92b7-e89637fc15ef">
<img width="382" alt="image" src="https://github.com/telegraf/telegraf/assets/18448635/477163ef-60d0-46d0-a75a-b80f22643b6a">

**Comment by MKRhere** [comment]
My answer there is still the answer.

> The bot has to be in the linked discussion group, and it can receive updates from the channel in the group chat. These messages will have `ctx.message.is_automatic_forward`, so you know these are the messages you should reply to. Replies to these messages will become comments on the channel post. [_source_](
https://github.com/telegraf/telegraf/discussions/1875#discussioncomment-7039911)

Telegram clients seem to bypass this requirement by under the hood making several calls to find the correct linked group to send messages to. Bots cannot do this.
